python_cmake_module/cmake/Modules/FindPythonExtra.cmake

```diff
@@ -128,6 +128,18 @@ if(PYTHONINTERP_FOUND)
     message(STATUS "Using PythonExtra_LIBRARIES: ${PythonExtra_LIBRARIES}")
   else()
     find_package(PythonLibs 3.5 REQUIRED)
+    if(WIN32 AND "${CMAKE_BUILD_TYPE} " STREQUAL "Debug ")
```

Comment:
Please remove the quotes and explicit variable expansion as well as the trailing space.